src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements GetFigureArea{
+    private double base1;
+    private double base2;
+    private double height;
+
+    @Override
+    public String toString() {

```

Comment:
According to the task description, you should not override the `toString()` method to 'draw' the figure. Instead, create a separate method to print out all information about the figure using `System.out.println()`.